cabal.project

```diff
@@ -38,6 +38,9 @@ test-show-details: direct
 -- Always write GHC env files, because they are needed for ghci.
 write-ghc-environment-files: always
 
+constraints:
+  io-classes-mtl < 0.1.2.0
```

Comment:
Can you add a comment to explain why this is necessary?

Reply:
It doesn't compile otherwise.  I think it's because of some changes to typeclasses that haven't yet percolated through all our dependencies.

Reply:
If you don't include it you get this error:

```
src/Control/Monad/Class/MonadSTM/Trans.hs:116:5: error: [GHC-54721]
    ‘writeTMVar’ is not a (visible) method of class ‘MonadSTM’
    |
116 |     writeTMVar     = ContTSTM .: writeTMVar
    |     ^^^^^^^^^^

src/Control/Monad/Class/MonadSTM/Trans.hs:194:5: error: [GHC-54721]
    ‘writeTMVar’ is not a (visible) method of class ‘MonadSTM’
    |
194 |     writeTMVar     = lift .: writeTMVar
    |     ^^^^^^^^^^

src/Control/Monad/Class/MonadSTM/Trans.hs:272:5: error: [GHC-54721]
    ‘writeTMVar’ is not a (visible) method of class ‘MonadSTM’
    |
272 |     writeTMVar     = lift .: writeTMVar
    |     ^^^^^^^^^^

src/Control/Monad/Class/MonadSTM/Trans.hs:350:5: error: [GHC-54721]
    ‘writeTMVar’ is not a (visible) method of class ‘MonadSTM’
    |
350 |     writeTMVar     = lift .: writeTMVar
    |     ^^^^^^^^^^

src/Control/Monad/Class/MonadSTM/Trans.hs:428:5: error: [GHC-54721]
    ‘writeTMVar’ is not a (visible) method of class ‘MonadSTM’
    |
428 |     writeTMVar     = lift .: writeTMVar
    |     ^^^^^^^^^^

src/Control/Monad/Class/MonadSTM/Trans.hs:506:5: error: [GHC-54721]
    ‘writeTMVar’ is not a (visible) method of class ‘MonadSTM’
    |
506 |     writeTMVar     = lift .: writeTMVar
    |     ^^^^^^^^^^

src/Control/Monad/Class/MonadSTM/Trans.hs:584:5: error: [GHC-54721]
    ‘writeTMVar’ is not a (visible) method of class ‘MonadSTM’
    |
584 |     writeTMVar     = lift .: writeTMVar
    |     ^^^^^^^^^^

src/Control/Monad/Class/MonadSTM/Trans.hs:662:5: error: [GHC-54721]
    ‘writeTMVar’ is not a (visible) method of class ‘MonadSTM’
    |
662 |     writeTMVar     = lift .: writeTMVar
    |     ^^^^^^^^^^
```

This is just changes in the definition of type classes in our dependencies that break other dependencies.  When all our other dependencies update accordingly we can remove this constraint.